Pods/Local Podspecs/MiniApp.podspec.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Do you think its a good idea to commit Pod files?
Please check https://guides.cocoapods.org/using/using-cocoapods.html#should-i-ignore-the-pods-directory-in-source-control

Reply:
Well, Cocoapods recommendation from the page you linked is :

> Whether or not you check in your Pods folder is up to you, as workflows vary from project to project. We recommend that you keep the Pods directory under source control, and don't add it to your .gitignore. But ultimately this decision is up to you

Personally I'm not advising to commit pods, because it's a lot of files that can be retrieved easily with pod install. The fact is this documentation page is not taking in account the case were you are working on a new pod. What happens if you don't commit your development pod files is that if one of the team members adds a new file in the pod structure, this file won't be added to the pod project structure of your teammates when they merge your changes... unless you pod update, what I totally missed ! So the answer is no, it's not a good idea, but it's not a bad idea either. I think I'll remove them from source control anyway and stick to the pod update way.

Reply:
You are right. We can skip this for now, but we should find a workaround for this.